library/src/main/java/com/getstream/sdk/chat/adapter/MediaAttachmentSelectedAdapter.java

```diff
@@ -75,6 +75,13 @@ public void bind(Attachment attachment, final OnItemClickListener listener) {
                     .solidColor(Color.BLACK)
                     .cornerRadii(cornerRadius, cornerRadius, cornerRadius, cornerRadius)
                     .build());
+
+            binding.ivMask.setShape(context, new DrawableBuilder()
```

Comment:
When I try to add a second file the dropdown is hidden

![image](https://user-images.githubusercontent.com/88735/66919746-3af1f300-f022-11e9-9b95-d612970d2ad8.png)

Reply:
Somehow I manage to do this on the simulator

![image](https://user-images.githubusercontent.com/88735/66920098-ed29ba80-f022-11e9-9fa7-57f97f51164e.png)

When this happens the image is selected forever

Reply:
This issue was fixed on this PR.
Please try this branch.